src/main/scala/fm/html/Html5CodeGen.scala

```diff
@@ -123,7 +123,7 @@ trait Html5 {
     
     val openBodyLines = Vector.newBuilder[String]
     
-    openBodyLines ++= (tag.required ++ tag.optional).map{ p: String => s"""if (${fixParamName(p)} ne null) ctx.append(" $p=\\""+h(${fixParamName(p)})+"\\"")""" }
+    openBodyLines ++= (tag.required ++ tag.optional).map{ p: String => s"""if (StringUtils.isNotBlank(${fixParamName(p)})) ctx.append(" $p=\\""+h(${fixParamName(p)})+"\\"")""" }
```

Comment:
mostly just trying to be consistent, I have no idea what your code is doing here